loader/valgrind.supp

```diff
@@ -0,0 +1,29 @@
+{
+   <leak from zai interceptor>
```

Comment:
What's that zai interceptor leak? Sounds like a mistake in our extension?

Reply:
It's really just `zend_hash_destroy(&zai_hook_static_inheritors)` missing in `zai_hook_mshutdown`.

Reply:
The fact that it's in the appsec suppression file probably just means "not our problem" for them :-)

Reply:
Sure :stuck_out_tongue:  But I'm also surprised it hasn't be reported by our ddtrace test suite :thinking: